src/apps/LTIMES_NOVIEW-Hip.cpp

```diff
@@ -70,16 +70,43 @@ void LTIMES_NOVIEW::runHipVariant(VariantID vid)
 
     LTIMES_NOVIEW_DATA_TEARDOWN_HIP;
 
+  } else if ( vid == Lambda_HIP ) {
+
+    LTIMES_NOVIEW_DATA_SETUP_HIP;
+
+    startTimer();
+    for (RepIndex_type irep = 0; irep < run_reps; ++irep) {
+
+      dim3 nthreads_per_block(num_m, num_g, 1);
+      dim3 nblocks(1, 1, num_z);
+
+      auto ltimes_noview_lambda = [=] __device__ (Index_type z, Index_type g, Index_type m) {
```

Comment:
@MrBurmark is there a reason why the implementation pattern is slightly different between Lambda_CUDA (define the lambda in the function call) and Lambda_HIP (use two local auto variables)? It appears to be done consistently across all the new variants you added.

Reply:
Yes, cuda triple chevron syntax can deduce the type of the lambda for the global function, but the hipLaunch function can't. So I have to put the lambda type in when I get the function pointer for hip but not for cuda.

Reply:
Gotcha. Thanks.

Reply:
I see this merged, but this begs the question of why it's not using either the launch call syntax or the triple chevron for both.  They both offer both, and triple chevron is measurably slower on both IIRC.

Reply:
Can you add an issue for this?

Reply:
@MrBurmark is this a request toadd an issue to @trws or me?